app/src/main/java/com/framgia/englishconversation/utils/binding/BindingUtils.java

```diff
@@ -240,4 +244,14 @@ public static void setTextHtml(TextView textView, String text) {
         }
         textView.setText(Html.fromHtml(text));
     }
+
+    @BindingAdapter({"bind:visiable"})
+    public static void setVisiable(View view, UserModel userModel) {
+        SharedPrefsApi prefsApi = SharedPrefsImpl.getInstance();
+        if (userModel.getEmail().equals(prefsApi.get(SharedPrefsKey.PREF_EMAIL, String.class))) {
+            view.setVisibility(View.VISIBLE);
+        }
```

Comment:
ERROR: '}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).